src/tools/compiletest/src/header.rs

```diff
@@ -113,6 +113,8 @@ pub struct TestProps {
     // testing harness and used when generating compilation
     // arguments. (In particular, it propagates to the aux-builds.)
     pub incremental_dir: Option<PathBuf>,
+    // If `true`, this test will use incremental compilation.
```

Comment:
Can we add some docs here (since this is basically where this is documented) about what 'scenario' of incr comp the test will be run under? In particular, I think we will ensure that the test runs each time starting with a clean incr. comp. dir and then any revisions will continue in the same directory (i.e., become "incr-patched" scenarios effectively).

Reply:
Of course!  I was intending to update the [rustc dev guide docs](https://rustc-dev-guide.rust-lang.org/tests/adding.html#other-header-commands) after this merged, but I understand that some people will only read one source or the other.